radixdlt/src/main/java/org/radix/api/services/InternalService.java

```diff
@@ -72,7 +72,7 @@ public Spammer(ECKeyPair owner, int iterations, int batching, int rate, int nonc
 			this.iterations = iterations;
 			this.rate = rate;
 			this.batching = Math.max(1, batching);
-			this.account = RadixAddress.from(universe, owner.getPublicKey());
+			this.account = new RadixAddress((byte) universe.getMagic(), owner.getPublicKey());
```

Comment:
Why was this changed to a constructor?

Reply:
@flotothemoon Java Common has no knowledge of a `Universe`, hence it cannot contain that constructor.

In `Swift` we could have written that constructor inside this repo (Core), by `extension`, don't know if that is possible in Java?

Reply:
I just meant an equivalent static constructor (that doesn't need to know about universe either), that is, something like `RadixAddress.from((byte) universe.getMagic(), owner.getPublicKey())`. Up to you.